### PR TITLE
TCD-2762: RDC Data

### PR DESCRIPTION
**Pull request by LoryApiFortress**
### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation fix (typos, incorrect content, missing content, etc.)

### Checklist

- [ ] I have read the [contributing](https://github.com/saucelabs/sauce-docs/blob/main/docs/contributing.md) document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: docs/insights/scope.md [hunk 1]
```diff
@@ -17,14 +17,14 @@ Sauce Labs has also published an [Insights API](/dev/api/insights) so you can bu
...
+| Trends           | Shows graphical visualizations of all tests. Applying filters to this view makes it easy to compare test outcomes for different variables, such as device browser version. See the [Trends](/insights/trends) page for detailed documentation.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     |
```
Comment (LoryApiFortress) [inline docs/insights/scope.md:24]:
```suggestion
| Trends           | Shows graphical visualizations of all tests. Applying filters to this view makes comparing test outcomes for different variables, such as the device browser version, manageable. See the [Trends](/insights/trends) page for detailed documentation.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     |
```

File: docs/insights/scope.md [hunk 2]
```diff
@@ -34,14 +34,14 @@ At the top of each page in the **Insights** interface is a set of properties by
...
+| ----------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Owner             | Show data for tests owned by one or more individuals, teams, or your whole organization. You will see different options here depending on your access and role within your organization or team.                                                                                                                                                                                                                                                               |
```
Comment (LoryApiFortress) [inline docs/insights/scope.md:39]:
```suggestion
| Owner             | Show data for tests owned by one or more individuals, teams, or your whole organization. Depending on your access and role in your organization or team, you will see different options here.                                                                                                                                                                                                                                                               |
```

File: docs/insights/scope.md [hunk 3]
```diff
@@ -34,14 +34,14 @@ At the top of each page in the **Insights** interface is a set of properties by
...
+| Tag               | Show data for tests or builds that are categorized using the specified tag(s).The **Tag** filter is not available for all sections of the Insights interface.                                                                                                                                                                                                                                                                                                  |
+| Time Period       | Show data for tests that executed within the specified time period. In the selection modal, choose the **Relative** tab to set a duration up through the current day and time. Choose the **Absolute** tab to set a specific window within start and end dates.<br/><br/>Times and dates are in the local time of the logged in user. If you are not seeing data you expect in that duration, consider whether the test was executed in a different time zone. |
```
Comment (LoryApiFortress) [inline docs/insights/scope.md:45]:
```suggestion
| Time Period       | Show data for tests that executed within the specified time period. In the selection modal, choose the **Relative** tab to set a duration up through the current day and time. Choose the **Absolute** tab to set a specific window between start and end dates.<br/><br/>Times and dates are in the local time of the logged-in user. If you do not see the data you expect in that duration, consider whether the test was executed in a different time zone. |
```

File: docs/insights/scope.md [hunk 4]
```diff
@@ -34,14 +34,14 @@ At the top of each page in the **Insights** interface is a set of properties by
...
+| Tag               | Show data for tests or builds that are categorized using the specified tag(s).The **Tag** filter is not available for all sections of the Insights interface.                                                                                                                                                                                                                                                                                                  |
+| Time Period       | Show data for tests that executed within the specified time period. In the selection modal, choose the **Relative** tab to set a duration up through the current day and time. Choose the **Absolute** tab to set a specific window between start and end dates.<br/><br/>Times and dates are in the local time of the logged-in user. If you do not see the data you expect in that duration, consider whether the test was executed in a different time zone. |
```
Comment (LoryApiFortress) [inline docs/insights/scope.md:45]:
```suggestion
| Time Period       | Show data for tests that were executed in the specified time period. In the selection modal, choose the **Relative** tab to set a duration up through the current day and time. Choose the **Absolute** tab to set a specific window between start and end dates.<br/><br/>Times and dates are in the local time of the logged-in user. If you do not see the data you expect in that duration, consider whether the test was executed in a different time zone. |
```

File: docs/insights/trends.md [hunk 5]
```diff
@@ -39,6 +39,10 @@ The Trends section of the Insights feature provides a variety of data visualizat
   </tr>
 </table>
 
+The above sections are available for Virtual Cloud (VDC) and Real Device (RDC).
+
+You can also filter the results by **Owner**, **Build**, **Operating System (OS)**, **Browser** (in VDC) or **Device** (in RDC), **Tag** and **Date Range**. The Date Range is set by default to 7 days, but options are also available: 15 minutes, 1 hour, 6 hours, 12 hours, 1 day, and 30 days, or you can define the Start and End Dates.
```
Comment (LoryApiFortress) [inline docs/insights/trends.md:44]:
```suggestion
You can also filter the results by **Owner**, **Build**, **Operating System (OS)**, **Browser** (in VDC) or **Device** (in RDC), **Tag** and **Date Range**. The Date Range is set by default to 7 days, but options are also available: 15 minutes, 1 hour, 6 hours, 12 hours, 1 day, and 30 days, or you can define start and end dates by switching to the **Absolute** tab.
```

**Comment by LoryApiFortress** [comment]
@thehannahm Can you please review the changes related to https://saucelabs.slack.com/archives/C04NZ6MGYKW/p1679346722332629? Thanks

**Review by aliciakrebs — APPROVED** [review]

**Review by thehannahm — APPROVED** [review]
Looks good!

**Pull request merged**